properties/basic.py

```diff
@@ -339,14 +339,19 @@ def error(self, instance, value, error_class=None, extra=''):
             prefix = prefix + ' of a {cls} instance'.format(
                 cls=instance.__class__.__name__,
             )
+        print_value = repr(value)
+        if len(print_value) > 107:
+            print_value = '{}  ...  {}'.format(
+                print_value[:50], print_value[-50:]
+            )
         message = (
-            '{prefix} must be {info}. A value of {val!r} {vtype!r} was '
-            'specified. {extra}'.format(
+            '{prefix} must be {info}. An invalid value of {val} {vtype} was '
+            'specified.{extra}'.format(
                 prefix=prefix,
                 info=self.info or 'corrected',
-                val=value,
+                val=print_value,
                 vtype=type(value),
-                extra=extra,
+                extra=' {}'.format(extra) if extra else '',
```

Comment:
This is fun. I guess it's a little overcomplicated, but I am really liking the clamping logic to avoid the message growing longer when it hits a max limit... Fine I guess :)

Reply:
Yeah, I'm on the same page with you. It is now a bit complicated and opinionated about how these errors look. But I also think printing out the entire invalid value is also arbitrary and not very useful.